.size-limit.js

```diff
@@ -28,7 +28,7 @@ module.exports = [
     name: 'The size of the @material-ui/core modules',
     webpack: true,
     path: 'packages/material-ui/build/index.js',
-    limit: '91.2 KB',
+    limit: '91.3 KB',
```

Comment:
Once we fully commit to forwardRef push it is probably a good idea to reexamine how much it actually cost. I feel like most of this is due to the local `displayName` "logic" and we can probably move this to the `getDisplayName` logic in `withStyles`.

Reply:
@eps1lon I think that we should take advantage of the v4 breaking change card it's offering us to aggressively try changes, then adjust based on the people & real life implications.